index.html

```diff
@@ -13226,12 +13227,12 @@ <h2>Excluding Elements from the Accessibility Tree</h2>
 	  <h2>Including Elements in the Accessibility Tree</h2>
           <p>If not already excluded from the accessibility tree per the rules above in <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a>, user agents MUST provide an <a>accessible object</a> in the <a>accessibility tree</a> for <abbr title="Document Object Model">DOM</abbr> <a>elements</a> that meet any of the following criteria:</p>
           <ul>
-            <li>Text nodes</li>
+            <li><a>Text nodes</a></li>
```

Comment:
This will just end up being a conflict when PR #1100 is merged.

```suggestion
            <li>Text nodes</li>
```